docs/sources/nodes.md

```diff
@@ -10,6 +10,34 @@ The TTL of the records can be set with the `external-dns.alpha.kubernetes.io/ttl
 Nodes marked as **Unschedulable** as per [core/v1/NodeSpec](https://pkg.go.dev/k8s.io/api@v0.31.1/core/v1#NodeSpec) are excluded.
 This avoid exposing Unhealthy, NotReady or SchedulingDisabled (cordon) nodes.
 
+## IPv6 Behavior
```

Comment:
There is a sentence in line 7 in this file, that should at least reference this heading.